src/scancode/cli.py

```diff
@@ -185,6 +186,11 @@ def validate_formats(ctx, param, value):
         raise click.BadParameter('Invalid template file: "%(value)s" does not exists or is not readable.' % locals())
     return value
 
+def ignore_files(ctx, param, value):
+    for file in value.split(','):
```

Comment:
Do not use a comma separated single option, instead use the proper click option that can accept `--ignore "abc*" --ignore "src/*/foobar` ...